### ISSUE TITLE
Aria label of session entries should have type of session at the end

### ISSUE THREAD

**Issue by isidorn** [microsoft/vscode#286374]
Right now the aria label seems to be TYPE LABEL. Instead it should be LABEL TYPE.
The practice we should follow is to always first announce the most important information to the user, and that is the label of the chat session

<img width="921" height="866" alt="Image" src="https://github.com/user-attachments/assets/cdded9cd-8b71-4944-974c-78ded2afbf36" />

@joshspicer @meganrogge

**Comment by meganrogge** [comment]
Confirmed with @accesswatch and @jooyoungseo that we want the type first, so we can close this
